install/installer/pkg/components/image-builder-mk3/deployment.go

```diff
@@ -163,7 +172,22 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 					TerminationGracePeriodSeconds: pointer.Int64(30),
 					Volumes:                       volumes,
 					InitContainers: []corev1.Container{
-						*common.InternalCAContainer(ctx),
+						{
+							Name:  "setup",
+							Image: ctx.ImageName(ctx.Config.Repository, Component, ctx.VersionManifest.Components.ImageBuilderMk3.Version), ImagePullPolicy: corev1.PullIfNotPresent,
+							Args: []string{
+								"setup",
+							},
+							SecurityContext: &corev1.SecurityContext{RunAsUser: pointer.Int64(0)},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "gitpod-ca-certificate",
+									SubPath:   "ca.crt",
+									MountPath: "/usr/local/share/ca-certificates/gitpod-ca.crt",
+								},
+							},
+							Env: common.ProxyEnv(&ctx.Config),
+						},
```

Comment:
same will need to be done for `image-builder-mk3-wsman/deployment.go` I think? See https://github.com/gitpod-io/gitpod/pull/16519

Reply:
done